misk-hibernate/src/main/kotlin/misk/hibernate/migrate/BulkShardMigrator.kt

```diff
@@ -206,10 +206,9 @@ class BulkShardMigrator<R : DbRoot<R>, C : DbChild<R, C>> private constructor(
             "SELECT id FROM $tableName WHERE $rootColumnName = :target AND id IN (:ids)"
         ).setParameter("target", targetRoot!!.id)
           .setParameterList("ids", sourceRecords.keys)
-          .list()
-          .stream()
-          .map { it as Long }
-          .collect(Collectors.toSet<Long>())
+            .resultList
```

Comment:
Is `resultList` short for `.list().stream()`?

Reply:
I believe so .. feels more _kotlinish_ than the clumsy java-style `.list().stream()`.